lib/richmond/rtfm.rb

```diff
@@ -104,15 +106,20 @@ def set_output_file?(line)
     end
 
     def assert_directory_exists(filename)
-      dirname = File.dirname(filename)
-      unless File.directory?(dirname)
-        FileUtils.mkdir_p(dirname)
-      end
+      #dirname = File.dirname(filename)
+      #unless File.directory?(dirname)
+        #FileUtils.mkdir_p(dirname)
+      #end
```

Comment:
I'm Toucan Sam when it comes to untested code... I looked at this code and suspected it wasn't under test.  I commented it out and the tests still passed.

It's always bits of code around the exterior of the flow, the type of code we always throw in because we know it's a good practice but a pain-in-the-butt to test.